src/scales/scale.time.js

```diff
@@ -360,8 +360,9 @@ module.exports = function(Chart) {
 		getLabelForIndex: function(index, datasetIndex) {
 			var me = this;
 			var label = me.chart.data.labels && index < me.chart.data.labels.length ? me.chart.data.labels[index] : '';
+			var firstData = me.chart.data.datasets[datasetIndex].data[0];
```

Comment:
I know that's the previous code, but why are we testing `data[0]` for null/object instead of `data[index]`?